dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -436,6 +436,22 @@ open class Jpql : JpqlDsl {
         return Expressions.minus(this.toExpression(), value.toExpression())
     }
 
+    /**
+     * Creates an expression that is enclosed in ceiling
+     */
+    @SinceJdsl("3.4.0")
+    fun <T : Number> ceiling(value: Expressionable<T>): Expression<T> {
```

Comment:
I don't think the `CEILING` is a primary function, so I think it would be better to have the function located below the `ABS`.

Reply:
I did. thanks